pom.xml

```diff
@@ -7,7 +7,7 @@
     <groupId>g-shock-team.robot</groupId>
     <artifactId>robot-shock-robot</artifactId>
     <packaging>pom</packaging>
-    <version>1.0-SNAPSHOT</version>
+    <version>${project.version}</version>
```

Comment:
We were told that project version should be constant. Not in properties. Not sure why. But check if mvn during build is not throwing warning about that.